addons/web/static/src/js/widgets/notification.js

```diff
@@ -53,22 +53,14 @@ var Notification = Widget.extend({
         this._closeCallback = params.onClose;
 
         if (this.type === 'danger') {
-            this.icon = 'fa-exclamation';
             this.className += ' bg-danger';
         } else if (this.type === 'warning') {
-            this.icon = 'fa-lightbulb-o';
             this.className += ' bg-warning';
         } else if (this.type === 'success') {
-            this.icon = 'fa-check';
             this.className += ' bg-success';
         } else if (this.type === 'info') {
-            this.icon = 'fa-info';
             this.className += ' bg-info';
         }
-
-        if (this.buttons && this.buttons.length) {
-            this.icon = 'fa-question-circle-o';
-        }
```

Comment:
The task is about making the toast looks like in the backend but you are modifying the framework code here... does it mean no backend toast was using those options maybe?

Reply:
No, some backend toast use also those options ...
So this PR remove the icons also in backend toast (when it's used)... But logically it sould be removed everywhere, if we dont like it 
@sbuhl  right ?

Reply:
@sbuhl What do you thing ?

Reply:
@bvr-odoo 
We want to get rid of them in the backend as well (I though it was already the case), so I'm ok with your removal.

I notify @aab-odoo on a precautionary basis (already got the green light from NGR).

cc @qsm-odoo

Reply:
@sbuhl Thanks for the ping, we'll adapt the code accordingly in our "wowl refactoring" branch (where we rewrite from scratch the webclient and among other things, the notification system).